README.md

```diff
@@ -55,6 +61,47 @@ in the form:
   }
 }
 ```
+- **Option 2:** Use a locally hosted Hyperledger Network (such as one from docker-compose) - [instructions](./docs/use_local_hyperledger.md)
+
+1. Clone this repository.
+2. Create an instance of the IBM Blockchain on your local Hyperledger network.
+3. Copy the credentials from the service into the file 'mycreds.json'.
+4. Make sure the key/value store only has values for your current network (See below).
+5. In users.js, change the ```affiliation``` in the registration request on **line 89**.  
+   Depending upon your registrar user's affiliation, it should look something like this:
+```
+ var registrationRequest = {
+                enrollmentID: enrollID,
+                affiliation: 'institution_a'    //change from "group1" to registrar's affiliation
+            };
+```
+6. Since the peers on the local network are listening using http, you need to make a few changes (see below).
+  - In [ws_part2.js](./utils/ws_part2.js), 
+    1. add ```var http = require('http');```
+    2. change ```https``` to ```http``` on **line 146**. It should look like this:
```

Comment:
You're right. There should be a way to handle that in ```process.env```, but I am not sure how to do that.